organization/_posts/2018-02-08-startup.md

```diff
@@ -0,0 +1,232 @@
+                    use some form of static analysers, mostly based on
+                    clang. How do we contribute back all those efforts
+                    so that everyone can profit from that?
+                -   Profilers. How do we share and compare profile data
+                    between different tools and experiments?
+                -   GVFS: git has become the defacto standard VCS in HEP
+                    and in the world. Is GVFS the next big revolution
+                    in the way we work? If yes, what are the
+                    implications / possibilities?
+            -   Becoming mainstream:
+                -   What are the OpenSource conferences / workshops we
+                    should try to have a more "active" presence?
+                    -   Python ones?
+                    -   FOSDEM?
+                -   Geek watch: how do we share information about the
+                    latests tools and practices? Mailing list?
+-   Thursday
+    -   Closing Plenary
+        -   Anticipate SKA contribution here: 20-30 minutes.
+
+CWP
+===
+
+-   ### General Matters and Roadmap
+    -   arXiv update a little delayed - announcement was late and
+        waiting for CMS resource plots with the correct axes labels
+        (MHS06s -> THS06s)
+        -   We now have all the pieces though, so this can be done
+            \~now.
+        -   Added 10 new signatures since last week.
+    -   Computing and Software for Big Science - discussed with Markus
+        Elsing. He did not think it would be a problem to update the
+        signatures after submission, but is confirming with other
+        editors and Springer.
+        -   The lead time for publication is >8 weeks, so we would be
+            ok to take signatures until the Naples workshop.
+-   ### Publication strategy for Individual WG Papers
+    -   #### Simulation
+        -   There is a final version of the long note; circulating
+            amongst the authors and calling for supporters.
+
+PyHEP Workshop
+==============
+
+-   Good progress with agenda and finding speakers.
+-   Sponsorship from PSF awaiting news (expect anytime within the next 2
+    weeks).
+
+Activity updates
+================
+
+Licensing
+---------
+-   No news.
+
+Training
+--------
+-   Pete Elmer is putting together an NSF proposal that might find some
+    HEP community training under the HSF umbrella.
+    -   Will be useful to discuss this further in Naples.
+
+Packaging
+---------
+-   Meeting 7 Feb:
+    [https://indico.cern.ch/event/700265/](https://indico.cern.ch/event/700265/)
+    -   Homing in on Use Cases Document.
+    -   Heard about SpackDev plans.
+    -   Report on use of Spack for FCC.
+        -   Further discussion in the next meeting
+-   Next meeting 21 Feb:
+    [https://indico.cern.ch/event/704402/](https://indico.cern.ch/event/704402/)
+
+GSoC
+----
+-   Proposals in, waiting for Google to sign off.
```

Comment:
Maybe say instead of "to sign off" that we are waiting for Google to confirm the CERN-HSF as an organisation also in the GSoC 2018 programme? And that this is due on Feb. 12th?